packages/jest-preset-mc-app/package.json

```diff
@@ -18,7 +18,6 @@
   "main": "./jest-preset.js",
   "module": "./jest-preset.js",
   "dependencies": {
-    "@babel/core": "7.9.0",
```

Comment:
It's a dev dependency apparently

<img width="608" alt="image" src="https://user-images.githubusercontent.com/1110551/80390185-647eb780-88ac-11ea-899f-a584dc81de1b.png">